src/main/resources/static/css/style.css

```diff
@@ -1,6 +1,11 @@
 .content {
     padding: 10px;
 }
+@media (max-width: 576px) {
```

Comment:
Usually you should only have one @media per screen size and all css for that size under this.

Reply:
Done.